app/assets/stylesheets/components/_markdown-editor.scss

```diff
@@ -82,6 +82,10 @@ $toolbar-button-size: 50px;
   .govuk-textarea {
     overflow: scroll;
   }
+
+  figure {
```

Comment:
I assume this is for within the govspeak rendering?

I'd suggest having it underneath the govspeak class rather than (or within) the markdown editor to indicate we're specifying styles on what govspeak outputs. Also can we target this by .image as per https://github.com/alphagov/govspeak/blob/master/README.md#images ?

Reply:
Sure - done.